sdk/src/main/java/io/radar/sdk/RadarBeaconManager.kt

```diff
@@ -617,12 +599,22 @@ internal class RadarBeaconManager(
         }
     }
 
-    internal fun isBluetoothSupported(context: Context): Boolean {
+    private fun isBluetoothSupported(context: Context): Boolean {
         if (!this::adapter.isInitialized) {
             adapter = BluetoothAdapter.getDefaultAdapter()
         }
 
         return context.packageManager.hasSystemFeature(PackageManager.FEATURE_BLUETOOTH) && adapter != null && adapter.bluetoothLeScanner != null
     }
 
+    private fun getScanSettings(): ScanSettings {
+        return ScanSettings.Builder()
+            .setScanMode(ScanSettings.SCAN_MODE_LOW_LATENCY)
```

Comment:
Just a heads up on how the docs talk about `LOW_LATENCY` and `LOW_POWER`:

`LOW_LATENCY`: recommended for foreground
`LOW_POWER` (default) = "enforced" if the scanning application is not in foreground

The "enforced" language is interesting.

https://developer.android.com/reference/android/bluetooth/le/ScanSettings#SCAN_MODE_LOW_LATENCY

We may want to peek at Battery Historian a bit over time.

Reply:
Updated to use `LOW_POWER` in background and `LOW_LATENCY` in foreground to be safe